ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFContactCollectionView)
+open class ContactCollectionView: UICollectionView {
+    struct Constants {
+        static let sideInset: CGFloat = 16.0
+        static let extraSmallContentContactHeight: CGFloat = 115.0
+        static let smallContentContactHeight: CGFloat = 117.0
+        static let mediumContentContactHeight: CGFloat = 118.0
+        static let largeContentContactHeight: CGFloat = 121.0
+        static let extraLargeContentContactHeight: CGFloat = 125.0
+        static let extraExtraLargeContentContactHeight: CGFloat = 129.0
+        static let extraExtraExtraLargeContentContactHeight: CGFloat = 135.0
```

Comment:
Instead of adding constants for each size category, we should extend the  enum.

extension UIContentSizeCategory {
  var contactHeight: CGFloat {
        switch self {
        case .extraSmall:
            return Constants.extraSmallContentContactHeight
        case .small:
            return Constants.smallContentContactHeight
        case .medium:
            return Constants.mediumContentContactHeight
        case .large:
            return Constants.largeContentContactHeight
        case .extraLarge:
            return Constants.extraLargeContentContactHeight
        case .extraExtraLarge:
            return Constants.extraExtraLargeContentContactHeight
        case .extraExtraExtraLarge:
            return Constants.extraExtraExtraLargeContentContactHeight
        default:
            return Constants.extraExtraExtraLargeContentContactHeight
    }
}

Later, you'll be able to do this to get the height:
UIApplication.shared.preferredContentSizeCategory.contactHeight

Reply:
Don't we still need to define the constant values for the height somewhere?

Reply:
No need if you follow my approach